src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -44,31 +44,45 @@ const collapsedStatuses = _.flow(
 )
 
 const statusCell = workflowStatuses => {
-  const { succeeded, failed, running, submitted } = collapsedStatuses(workflowStatuses)
+  const { succeeded, failed, running, submitted, abort } = collapsedStatuses(workflowStatuses)
 
   return h(TooltipTrigger, {
     side: 'bottom',
     type: 'light',
     content: table({ style: { margin: '0.5rem' } }, [
       tbody({}, [
-        tr({}, [
+        submitted ? tr({}, [
```

Comment:
This is where we'd use `&&` instead of the ternary. You could say `!!submitted` as well, for clarity.